apps/challenges/models.py

```diff
@@ -220,3 +220,15 @@ class ChallengeConfiguration(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_zip_configuration'
+
+
+class StarChallenge(TimeStampedModel):
```

Comment:
Just a curiosity question here... :sweat_smile: that there can be many users who will star and then unstar, what can a challenge host do by knowing about them ?